packages/whale-api-client/__tests__/api/rpc.test.ts

```diff
@@ -68,4 +68,16 @@ describe('whitelisted rpc methods', () => {
     expect(block.hash.length).toStrictEqual(64)
     expect(Array.isArray(block.tx)).toStrictEqual(true)
   })
+
+  it('should rpc.call(getrawtransaction)', async () => {
+    await container.generate(1)
+
+    const hash = await client.rpc.call<string>('getblockhash', [1], 'number')
+    const block = await client.rpc.call<blockchain.Block<blockchain.Transaction>>('getblock', [hash], 'number')
+    const hex = await client.rpc.call<string>('getrawtransaction', [block.tx[0]], 'number')
```

Comment:
Added another test inside the rpc.controller.spec.ts

Reply:
> You should add another test to call it verbosely since your use case requires it.

In the use case described in the issue, this RPC should be called verbose;y since the raw hex output won't be of any use to the consumer. And the fields that are required should be asserted.

https://jellyfishsdk.com/node/jellyfish/api/rawtx#getrawtransaction

```diff
   it('should rpc.call(getrawtransaction)', async () => {
     await client.rpc.call<string>('getrawtransaction', [block.tx[0]], 'number')
+  it('should rpc.call(getrawtransaction) verbose=true', async () => {
+    await client.rpc.call<string>('getrawtransaction', [block.tx[0], true], 'number')
```

Reply:
Nope, not a new endpoint. Your test result shouldn't return a hex string, it should return structured data.

Reply:
Ok got your point. But the rpc call for getblockhash returns also just the id of the block. 
But I like the rawTx endpoint way more. Looks more natural. I will change the response to structured data there.

Reply:
TBH: rawtx.controller.ts Send does the same. Also returns just the transactionId after commiting the tx to the chain!?

Reply:
The current design of `rawtx` deal exclusively with the transaction in raw form (aka raw encoded hex), you send a transaction serialized as "raw hex" and you get the transaction id as hex back. Hence the dissimilarity, and the need for it to stay specific to the encoded hex rather than structured data is to keep the API design consistent. (TBH, I personally hate REST API design, but since it's done like this thus far, we have to stick to it for the sake of consistency)

The reason why I requested the tests you have written so far to test the intended outcome is due to the issue written in #1718. Since the intent was clear and what was required to "fix" the issue was obvious, the tests written should reflect how one would use the newly exposed RPC endpoint. Aka dogfooding your own API with tests intended by the consumer.
> https://github.com/JellyfishSDK/jellyfish/blob/main/CONTRIBUTING.md#end-to-end-testing

Reply:
But I have added the tests exactly I want to use it. Sending the transaction id and get the raw tx (as hex) returned from the api.

Reply:
Ahh, okay! I'm starting to see what you're trying to achieve. Let me approve the CI runs, I will review this PR when the test passes.